composer.json

```diff
@@ -59,6 +59,7 @@
         "drupal/profile": "^1.3",
         "drupal/redirect": "^1.6",
         "drupal/rdf": "^2.1",
+        "drupal/rabbit_hole": "^1.0@beta",
```

Comment:
Please use 2.0 version instead as it `Recommended by the project’s maintainer`.
The first one only works for few entity types and given we have a dozen custom entity types it would be more justified to use second one